src/Kmd.Logic.Gateway.Automation/PreValidation/PreValidationBase.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+
+namespace Kmd.Logic.Gateway.Automation
+{
+    public class PreValidationBase
```

Comment:
I would suggest `EntityPreValidationBase`

Reply:
Changed to EntityPreValidationBase